Bio/PDB/PSEA.py

```diff
@@ -30,12 +31,22 @@ def run_psea(fname):
     Note that P-SEA will write an output file in the current directory using
     the input filename with extension ".sea".
 
-    Note that P-SEA will write output to the terminal while run.
+    Note that P-SEA will not write output to the terminal while run unless
+     verbose is set to True.
     """
-    subprocess.call(["psea", fname])
     last = fname.split("/")[-1]
     base = last.split(".")[0]
-    return base + ".sea"
+    cmd = ["psea", fname]
+
+    p = subprocess.run(cmd, capture_output=True)
+
+    if verbose:
+        print(p.stdout.decode("utf-8"))
+
+    if not p.stderr.decode("utf-8") and exists(base + ".sea"):
```

Comment:
Just for (paranoid) safety, I'd do `if not p.stderr.strip()`, in case there is some straggler whitespace in there.